doc/ref/configuration/master.rst

```diff
@@ -4264,6 +4277,19 @@ is equivalent to this static configuration:
       prod:
         - /srv/prod/pillar
 
+As of 3007.0, the ``pillar_roots`` option is dynamically expanded on each use,
```

Comment:
3008.0

Reply:
Doh, updated one reference but missed this one.